StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -11,16 +11,18 @@
 @_spi(STP) import StripeUICore
 import UIKit
 
-class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewController {
+class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewControllerProtocol, PaymentSheetViewControllerProtocol {
     var selectedPaymentOption: PaymentSheet.PaymentOption?
     var selectedPaymentMethodType: PaymentSheet.PaymentMethodType?
-    weak var delegate: FlowControllerViewControllerDelegate?
     let loadResult: PaymentSheetLoader.LoadResult
     let configuration: PaymentSheet.Configuration
     var intent: Intent {
         return loadResult.intent
     }
     var error: Error?
+    let isFlowController: Bool
```

Comment:
Does it make sense to attach this to the load result?

Reply:
I don't think so, it's not something that's loaded.